src/garage/sampler/vec_env_executor.py

```diff
@@ -16,43 +22,77 @@ def __init__(self, envs, max_path_length):
         self.max_path_length = max_path_length
 
     def step(self, action_n):
-        """Step all environments using the provided actions."""
+        """Step all environments using the provided actions.
+
+        Args:
+            action_n (np.ndarray): Array of actions.
+
+        Returns:
+            tuple: Tuple containing:
+                * observations (np.ndarray)
+                * rewards (np.ndarray)
+                * dones (np.ndarray): The done signal from the environment.
+                * completes (np.ndarray): Episode end (time limit reached or
```

Comment:
so it's a little weird that this used to implicitly implement gym.Env.step  and now does not (because of completes). Is it important that completes be returned in this tuple, or can it be put into env_infos?